.github/workflows/on-push.yml

```diff
@@ -5,11 +5,7 @@ on:
     branches:
       - '**'
     tags-ignore:
-      - '**'
-    paths-ignore:         # Don't trigger on files that are updated by the CI
```

Comment:
Why do we no longer need the paths-ignore?

Reply:
It's not working. I have a discussion on support going. In the meantime, I'll remove it.